include/grpc/support/time.h

```diff
@@ -55,9 +55,9 @@ typedef struct gpr_timespec {
 
 /** Time constants. */
 GPRAPI gpr_timespec
-gpr_time_0(gpr_clock_type type); /** The zero time interval. */
-GPRAPI gpr_timespec gpr_inf_future(gpr_clock_type type); /** The far future */
-GPRAPI gpr_timespec gpr_inf_past(gpr_clock_type type);   /** The far past. */
+gpr_time_0(gpr_clock_type type); /**< The zero time interval. */
```

Comment:
How about having `/** */` like other? Comments need to move to the front then.

Reply:
Done.